### PR TITLE
fixes bug #4

### PR DESCRIPTION
**Pull request by Xenomer**
the user will now be removed from the correct list when leaving from reserve

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -196,35 +196,53 @@ router.get('/deleteParticipant', function(req, res) {
...
+            reservedParticipantsCache.splice(index, 1);
+
+            // remove from usr locally
+            var reservedEventsCache = JSON.parse(removedUserData.reservedEvents);
+            var index2 = reservedEventsCache.indexOf(req.query.id);
+            reservedEventsCache.splice(index2, 1);
+
+            collection.update({ _id: req.query.id }, {$set: { reservedParticipants: JSON.stringify(reservedParticipantsCache) }});
+            usrCollection.update({ _id: removedUserData._id }, {$set: { reservedEvents: JSON.stringify(reservedEventsCache) }});
+            res.send("removed from reserve");
```
Comment (kohvakka) [inline routes/index.js:241]: The client doesn't know what this response means. You should stick to old responses or change the client!

File: routes/index.js [hunk 2]
```diff
@@ -196,35 +196,53 @@ router.get('/deleteParticipant', function(req, res) {
...
+            docsCache = JSON.parse(docsCache.participants);
+            reservedParticipantsCache = JSON.parse(docs.reservedParticipants);
+            docsCache.splice(index,1);
+            if(reservedParticipantsCache.length > 0) {
+              usrCollection.findOne({ _id: reservedParticipantsCache[0] },{},function(e,usrDocs){
+                if(e) {
+                  console.log(e);
+                  res.send("That user doesn't exist or the database is down");
+                }
+                console.log(usrDocs);
```
Comment (kohvakka) [inline routes/index.js]: These conhsole.logs should probably be removed

**Review by kohvakka — CHANGES_REQUESTED** [review]
It's working nicely, but there are these few things that you could checkout!

**Comment by Xenomer** [comment]
modified pull request with proposed logging changes and [created another ](https://github.com/TheKooxd/prophet-client/pull/5) for proposed client changes

**Review by kohvakka — APPROVED** [review]
All clear, merging the pull request to branch dev_0.1.0

**Pull request merged**